index.js

```diff
@@ -43,10 +43,13 @@ class SortedList {
   }
 
   avg() {
+    if(this.items.length === 0){
+      error
```

Comment:
avg() {
    // !this.length
    if (this.items.length === 0) {
      throw new Error('EmptySortedList');
    }
    // we get average simple as dividing sum with the number of elements in the array (which is the length of the array)
    return this.sum() / this.length;
  }